### ISSUE TITLE
Cannot install 1.2.3 python bindings on python 3.6 on MacOS Catalina

### ISSUE THREAD

**Issue by pmezard** [rickardp/splitstream#11]
Hello,

I can install 1.2.2 python bindings but the latest 1.2.3 release fails with:
```
Running setup.py install for splitstream ... error
    Complete output from command /some/path/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/private/var/folders/_r/vnrbghg13992b97hxz1tmh6r0000gn/T/pip-install-tvcq5a55/splitstream/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/_r/vnrbghg13992b97hxz1tmh6r0000gn/T/pip-record-h8p0iiv5/install-record.txt --single-version-externally-managed --compile --install-headers /some/path/include/site/python3.6/splitstream:
    running install
    running build
    running build_ext
    building 'splitstream' extension
    creating build
    creating build/temp.macosx-10.15-x86_64-3.6
    creating build/temp.macosx-10.15-x86_64-3.6/src
    creating build/temp.macosx-10.15-x86_64-3.6/src/python
    /usr/bin/clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -I/some/path/include -I/opt/local/Library/Frameworks/Python.framework/Versions/3.6/include/python3.6m -c src/python/splitstream_py.c -o build/temp.macosx-10.15-x86_64-3.6/src/python/splitstream_py.o
    src/python/splitstream_py.c:21:10: fatal error: '../splitstream.h' file not found
    #include "../splitstream.h"
             ^~~~~~~~~~~~~~~~~~
    1 error generated.
    error: command '/usr/bin/clang' failed with exit status 1
```

MacOS 10.15.5.
Python version (macports):
```
$ python
Python 3.6.11 (default, Jun 29 2020, 13:22:29) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```

Tell me if you need additional information.

**Comment by rickardp** [comment]
I switched the build pipelines from Travis to GitHub Actions. I could have broken something. I will investigate. Thanks for reporting.

**Comment by rickardp** [comment]
It seems when building from pipelines, I needed to add a MANIFEST.in file with the headers (despite the headers being specified in setup.py). Seems to be a known issue, but I am not sure why it worked before on my MacBook. Anyway, version 1.2.4 (code-wise identical to 1.2.3) fixes the issue and I yanked 1.2.3 to remove any confusion.

**Comment by pmezard** [comment]
Thanks!
